Yoast/wordpress-seo
# Add object id and object type index

## Summary


This PR can be summarized in the following changelog entry:

* Adds a missing index to the new indexables table.

## Test instructions

This PR can be tested by following these steps:

* Open any page.
* You should now have an index on object id and object type.

## UI changes

## Documentation

## Quality assurance


Fixes #15004
